### DISCUSSION TITLE
No audible sound using certain apps on M1 Mac?

### DISCUSSION THREAD

**Discussion by BigBillete** [ExistentialAudio/BlackHole#431] [Q&A]
Hello, I recently installed and have been using black hole to capture internal audio on my M1 Mac mini, and its been working great with no real issues, I planned on using it to stream some of the GTA titles now available to play on M1 Macs since apple has allowed compatibility with certain IOS apps/games. Now my issue is whenever I launch any of the IOS GTA games, the audio decibel meter for the blackhole multi output device I had to create in the Audio MIDI setup application, in OBS shows that it is receiving sound from the application, but I can't hear anything. Like I said I had everything set up properly and can hear audio from spotify, safari, Minecraft, but nothing from the GTA games. Im not sure what the issue is, if I swap between audio interfaces in the sound preference in settings on my Mac, I can switch from external headphones and the blackhole multi output device and can only hear the game when its selected on external headphones. Not sure if its just a bug from the game or maybe I have to change a setting somewhere on my computer or in the Audio MIDI setup application. sorry if its difficult to understand ill try to make a YouTube video explaining my situation if any one would be willing to help me out. Thanks

**Comment by devinroth** [comment]
If it’s working normally with some apps and not others it means those apps don’t support multi outputs. I have a short list in the faq.

Devin

**Reply by BigBillete** [reply]
Oh ok makes sense thanks a lot!
